dgv/metrics/task_function_test.py

```diff
@@ -0,0 +1,285 @@
+            ' https://www.data.gouv.fr/en/datasets/etat-civil-prenoms-des-nouveau-nes/?hbjkwxz=hbjkwxz HTTP/2.0"',
+            "expected_output": (
+                "etat-civil-prenoms-des-nouveau-nes",
+                "datasets",
+                "en",
+            ),  # /en/datasets/$SLUG
+        },
+        {
+            "log": "2025-01-22T00:03:20.860506+01:00 slb-03 haproxy[2021969]: 127.0.0.1:6080 [22/Jan/2025:00:03:20.010]"
+            ' DATAGOUVFR_RGS~ DATAGOUVFR_NEWINFRA/datawrk-03 0/0/2/848/+850 200 +1365 - - --NR 123/76/3/1/0 0/0 "GET'
+            ' /fr/datasets/6322e99e12175f7eb26ff465/ HTTP/1.1"',
+            "expected_output": (
+                "6322e99e12175f7eb26ff465",
+                "datasets",
+                "fr",
+            ),  # /fr/datasets/$ID
+        },
+        {
+            "log": "2025-01-22T00:02:27.081346+01:00 slb-03 haproxy[2021969]: 127.0.0.1:51723 [22/Jan/2025:00:02:26.607]"
+            ' DATAGOUVFR_RGS~ DATAGOUVFR_NEWINFRA/dataweb-06 0/0/1/472/+473 200 +4262 - - --VN 154/114/4/4/0 0/0 "GET'
+            ' https://www.data.gouv.fr/fr/datasets/?q=ancv HTTP/2.0"',
+            "expected_output": (
+                "?q=ancv",
+                "datasets",
+                "fr",
+            ),  # /fr/datasets/ with no ID
+        },
+        {
+            "log": "2025-01-22T00:04:13.851643+01:00 slb-03 haproxy[2021969]: 127.0.0.1:11480 [22/Jan/2025:00:04:13.551]"
+            ' DATAGOUVFR_RGS~ DATAGOUVFR_NEWINFRA/dataweb-05 0/0/3/296/+299 200 +7157 - - --NR 132/95/4/3/0 0/0 "GET'
+            ' /es/organizations/commune-de-rosenau/ HTTP/1.1"',
+            "expected_output": (
+                "commune-de-rosenau",
+                "organizations",
+                "es",
+            ),  # /es/organizations/$SLUG
+        },
+        {
+            "log": "2025-01-22T06:59:08.378845+01:00 slb-04 haproxy[2624750]: 127.0.0.1:27337 [22/Jan/2025:06:59:07.709]"
+            ' DATAGOUVFR_RGS~ DATAGOUVFR_NEWINFRA/dataweb-05 0/0/1/667/+668 200 +14397 - - --NR 102/79/3/2/0 0/0 "GET'
+            ' https://www.data.gouv.fr/fr/organizations/534fff94a3a7292c64a77fc1/ HTTP/2.0"',
+            "expected_output": (
+                "534fff94a3a7292c64a77fc1",
+                "organizations",
+                "fr",
+            ),  # /es/organizations/$ID
+        },
+        {
+            "log": "./haproxy-22012025-04.log:397163:2025-01-22T05:02:48.999363+01:00 slb-04 haproxy[2624750]: 127.0.0.1:43312 [22/Jan/2025:05:02:48.961]"
+            ' DATAGOUVFR_RGS~ DATAGOUVFR_NEWINFRA/datawrk-03 0/0/2/36/+38 200 +14164 - - --NN 114/102/5/2/0 0/0 "GET'
+            ' /api/1/organizations/580e1e1388ee385e8e13e4cb/datasets/ HTTP/1.1"',
+            "expected_output": (
+                "580e1e1388ee385e8e13e4cb",
+                "organizations",
+                "api1",
+            ),  # /es/organizations/$ID
```

Comment:
Am I getting this right?
```suggestion
            ),  # /api/1/organizations/$ID
```

Reply:
Yes 😅

Reply:
Btw do you feel like the comments add any value or not?

Reply:
Yes it nice to have them :)